environmentManager/index.js

```diff
@@ -47,6 +48,10 @@ class EnvironmentManager {
     return this._environments[authToken].factory;
   }
   
+  getVersion() {
+    return this._environments[this.getAuthTokens()[0]].factory.settings.sdkVersion;
```

Comment:
`.sdkVersion` --> `.version` ?

Reply:
it is `sdkVersion`, because this method is used to replace this line
`const parts = sdk.settings.sdkVersion.split('-');`
with 
`const parts = environmentManager.getVersion().split('-');`